Modules/_datetimemodule.c

```diff
@@ -84,33 +84,33 @@ class datetime.IsoCalendarDate "PyDateTime_IsoCalendarDate *" "&PyDateTime_IsoCa
 /* Date accessors for date and datetime. */
 #define SET_YEAR(o, v)          (((o)->data[0] = ((v) & 0xff00) >> 8), \
                  ((o)->data[1] = ((v) & 0x00ff)))
-#define SET_MONTH(o, v)         (PyDateTime_GET_MONTH(o) = (v))
-#define SET_DAY(o, v)           (PyDateTime_GET_DAY(o) = (v))
+#define SET_MONTH(o, v)         ((o)->data[2] = (v))
```

Comment:
Why do we need to change these? Is the change to `PyDateTime_GET_MONTH` et al a breaking change?

Reply:
This PR is a breaking change on purpose: it disallows using a macro **getting** a member to **set** a member, exactly like SET_MONTH().